### PR TITLE
Feat: Added LBD and UBD scenarios and Cyclones

### PR DESCRIPTION
**Pull request by johnorcutt**
Added UPB, LBD and cyclones. Updated the OpenAPI generated code using the latest swagger spec.

### CODE REVIEW

File: clients/python/sust/api/__init__.py [hunk 1]
```diff
@@ -1,5 +0,0 @@
-from .climate_explorer import (
```
Comment (bcwaldon) [inline clients/python/sust/api/__init__.py:1]: Add this file back in. I haven't excluded it from the area of effect from the generation process yet

File: clients/python/sust/api/climate_explorer.py [hunk 2]
```diff
@@ -8,13 +8,20 @@
...
     SSP245 = 'ssp245'
+    SSP245_ubd = "ssp245_ubd"
+    SSP585_lbd = "ssp585_lbd"
     SSP585 = 'ssp585'
+    SSP585_ubd = "ssp585_ubd"
 
 
 @dataclass(frozen=True)
 class RiskTypes:
+    CYCLONE = "cyclones"
```
Comment (bcwaldon) [inline clients/python/sust/api/climate_explorer.py:24]: Let's preserve the pluralization here

File: clients/python/sust/api/climate_explorer.py [hunk 3]
```diff
@@ -126,7 +131,8 @@ def physical_risk_summary(self, scenario=None):
             req_kwargs['scenario'] = scenario
 
         it = self._client._paginated_openapi_request('portfolios_datasets_physical_summary_list', (self._portfolio['portfolio_name'],), req_kwargs)
-
+        print("#$%$#"*100)
```
Comment (bcwaldon) [inline clients/python/sust/api/climate_explorer.py:134]: Please remove debug lines.

**Review by bcwaldon — CHANGES_REQUESTED** [review]

File: clients/python/sust/api/__init__.py (hunk 1 above)
Comment (johnorcutt) [reply clients/python/sust/api/__init__.py:1]: Yep, that was removed by the OpenAPI generator.

**Pull request merged**